desktop-cosmic/cosmic-comp/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=cosmic-comp
+PKGSEC=COSMIC
+PKGDEP="gcc-runtime libdisplay-info libinput libxkbcommon mesa pixman seatd \
+        systemd"
+BUILDDEP="fontconfig libdisplay-info libinput libxcb mesa pixman seatd \
```

Comment:
Ditto above for questionable build-time dependency.

Reply:
Also note that both `PKGDEP` and `BUILDDEP` are installed during build-time, so no need to specify twice.

Reply:
Same for many packages below.